manifests/rally-core-addon.yml

```diff
@@ -2,6 +2,7 @@ description: Rally Core Add-on
 repo-prefix: rallycoreaddon
 active: true
 private-repo: false
+branch: release
```

Comment:
I think this will work. We may want to test on ship-it today, so we know.

Reply:
Thanks. I'll cut a new release right now.